aws_cloudtrail_rules/aws_root_activity.py

```diff
@@ -5,6 +5,7 @@
 
 def rule(event):
     return (event['userIdentity'].get('type') == 'Root' and
+            event.get('errorMessage') is None and
             event['userIdentity'].get('invokedBy') is None and
             event['eventType'] != 'AwsServiceEvent' and
             event['eventName'] not in EVENT_ALLOW_LIST)
```

Comment:
Just my preference: I might prefer using `.get()` on all these accesses for consistency.